dask_cloudprovider/cloudprovider.yaml

```
@@ -119,6 +119,18 @@ cloudprovider:
     docker_image: "daskdev/dask:latest" # docker image to use
     bootstrap: true # It is assumed that the OS image does not have Docker and needs bootstrapping. Set this to false if using a custom image with Docker already installed.
 
+  ibm:
+    api_key: null
+    image: "daskdev/dask:latest"
```

We probably need to do this everywhere, but this image is no longer maintained and we should be using the GitHub hosted ones.

```suggestion
    image: "ghcr.io/dask/dask:latest"
```